apps/dashboard/lib/trpc/routers/utils/granularity.ts

```diff
@@ -44,6 +44,22 @@ export type TimeseriesConfig<TContext extends TimeseriesContext> = {
   context: TContext;
 };
 
+export const TIMESERIES_GRANULARITIES = {
+  perMinute: { ms: MINUTE_IN_MS, format: "h:mm:ssa" },
+  per5Minutes: { ms: 5 * MINUTE_IN_MS, format: "h:mma" },
+  per15Minutes: { ms: 15 * MINUTE_IN_MS, format: "h:mma" },
+  per30Minutes: { ms: 30 * MINUTE_IN_MS, format: "h:mma" },
+  perHour: { ms: HOUR_IN_MS, format: "MMM d h:mma" },
+  per2Hours: { ms: 2 * HOUR_IN_MS, format: "MMM d h:mma" },
+  per4Hours: { ms: 4 * HOUR_IN_MS, format: "MMM d h:mma" },
+  per6Hours: { ms: 6 * HOUR_IN_MS, format: "MMM d h:mma" },
+  per12Hours: { ms: 12 * HOUR_IN_MS, format: "MMM d h:mma" },
+  perDay: { ms: DAY_IN_MS, format: "MMM d" },
+  per3Days: { ms: 3 * DAY_IN_MS, format: "MMM d" },
+  perWeek: { ms: 7 * DAY_IN_MS, format: "MMM d" },
+  perMonth: { ms: 30 * DAY_IN_MS, format: "MMM yy" },
+} as const;
```

Comment:
@perkinsjr I think as you said we can enforce this granularity on every chart now. 🫡